test/appsignal/probes/erlang_probe_test.exs

```diff
@@ -8,7 +8,8 @@ defmodule Appsignal.Probes.ErlangProbeTest do
     # from this test
     Appsignal.Probes.unregister(:erlang)
 
-    [fake_appsignal: start_supervised!(FakeAppsignal)]
+    start_supervised!(FakeAppsignal)
```

Comment:
We started explicitly passing these pids around to make sure we were always talking about the same process here, which relieved some problems with tests taking a while to tear down. I’d prefer not having to do that and referring to the process’s name (`FakeAppsignal`), like in this change. 

If this change makes the tests more flaky, that could be the cause. If not, I’m +1 on this. :)

Reply:
Oh, thanks for the context @jeffkreeftmeijer! The tests in the PR have been quite flaky -- although not sure if _more_ flaky than usual. I'll try reverting that and see if that gets a build to pass 😅